src/compiler/utilities.ts

```diff
@@ -2414,6 +2414,27 @@ namespace ts {
         return output;
     }
 
+    export const stringify: (value: any) => string = JSON && JSON.stringify ? JSON.stringify : function stringify(value: any): string {
+        /* tslint:disable:no-null */
+        return value == null ? "null"

```

Comment:
Actually it should be `value === null`, as that's how I have it in the experimentalTransforms branch. This was a typo introduced while trying to work out how to disable a tslint rule :)